app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -574,6 +575,7 @@ def test_end_experiment_in_kinto_deletes_experiment(self):
             experiment.changes.filter(
                 old_publish_status=NimbusExperiment.PublishStatus.APPROVED,
                 new_publish_status=NimbusExperiment.PublishStatus.WAITING,
+                message="Deleted from Kinto",
```

Comment:
Just noticed some tests referring to literals and some referring to the constants, dunno if you wanna make it all one or the other 🤷‍♀️

Reply:
Yeah, I'll fix that - should be all consistent